src/Wizard/Services/ConnectivityService.ts

```diff
@@ -0,0 +1,44 @@
+import { Connectivity } from '../Connectivity/Connectivity';
+import { TestType } from '../Models/ConnectivityTest';
+import { inject, injectable, injectAll } from 'tsyringe';
+import { logger } from 'src/Utils';
+import { URL } from 'url';
+
+export const ConnectivityUrls = Symbol('ConnectivityUrls');
+
+@injectable()
+export class ConnectivityAnalyzer {
```

Comment:
Extract interface to meet DI https://github.com/NeuraLegion/nexploit-cli/pull/135#discussion_r568042523